builders/manage/sudo/pause-transactions.md

```diff
@@ -0,0 +1,108 @@
+    origin: OriginFor<T>,
+    dest: AccountIdLookupOf<T>,
+    #[pallet::compact] value: T::Balance,
+) -> DispatchResult {
+    // Code
+    Ok(())
+}
+```
+
+## Pausing Transactions {: #pausing-transactions }
+
+As you know, the Sudo account can perform privileged functions, such as pausing and unpausing transactions. To pause a transaction, navigate to the **Developer** tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, you have not associated the Sudo account with Polkadot.js Apps. Make sure that your [Sudo account is injected by your wallet and connected to Polkadot.js Apps](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank}. Then, take the following steps:
+
+1. Select the **txPause** module
+2. Select the **pause** method
+3. Insert the **module** name that contains the transaction that will be paused.
+4. Insert the **transaction** name that will be paused.
+5. Press **Submit Sudo** and confirm the transaction in the resulting pop-up
+
+In this example, the transaction paused is `transfer_allow_death` from the `Balances` module:
+
+![Pause transaction](/images/builders/manage/sudo/pause-transactions/pause-transactions-2.webp)
+
+To verify that the transaction has been effectively paused, try executing it. You should get an error.
+
+![Check that the transaction is paused](/images/builders/manage/sudo/pause-transactions/pause-transactions-3.webp)
+
+!!! warning
+    The `pause` transaction doesn't verify the module or transaction names and is case-sensitive, so any misspelling will go unnoticed, and the transaction will execute successfully. You should always verify that the transaction has been effectively paused.
+
+## Unpausing Transactions {: #unpausing-transactions }
+
+To unpause a transaction and return it to normal operation, navigate to the **Developer** tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, you have not associated the Sudo account with Polkadot.js Apps. Make sure that your [Sudo account is injected by your wallet and connected to Polkadot.js Apps](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank}. Then, take the following steps:
+
+1. Select the **txPause** module
+2. Select the **unpause** method
+3. Insert the **module** name that contains the transaction that will be unpaused.
+4. Insert the **transaction** name that will be unpaused.
+5. Press **Submit Sudo** and confirm the transaction in the resulting pop-up
+
+In this example, the transaction to unpause is `transfer_allow_death` from the `Balances` module:
+
+![Unpause transaction](/images/builders/manage/sudo/pause-transactions/pause-transactions-4.webp)
+
+The `unpause` transaction executes successfully only if the module and transaction parameters have been previously paused; otherwise, It fails. After the successful unpausing, the transaction can be called and executed again.
```

Comment:
```suggestion
The `unpause` transaction executes successfully only if the module and transaction parameters have been previously paused; otherwise, it fails. After the successful unpausing, the transaction can be called and executed again.
```